### PR TITLE
Feature/null safety

### LINKED ISSUE #33: Release a null-safe version
Please create a nullsafe prerelease version of this package

### PR DESCRIPTION
**Pull request by felix-barz-brickmakers**
Closes #33 

- The first commit does the actual migration to nullsafety
- The second commit fixes the example to use embedding v2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [1.0.0-nullsafety.0]
```
Comment (lazylazyllama) [inline CHANGELOG.md:1]: As null safety is in flutter stable, I think the "-nullsafety.0" suffix is not necessary anymore

Comment (felix-barz-brickmakers) [reply CHANGELOG.md:1]: fixed

Comment (felix-barz-brickmakers) [reply CHANGELOG.md:1]: @victorbonnet

**Review by victorbonnet — APPROVED** [review]

**Comment by victorbonnet** [comment]
Thanks @felix-barz-brickmakers and @lazylazyllama for the PR.
It has been merged and published.

**Pull request merged**